src/invoice_details.rs

```diff
@@ -1,5 +1,7 @@
-use crate::amount::Amount;
+use crate::amount::{Amount, ToAmount};
 
+use crate::ExchangeRate;
+use breez_sdk_core::LNInvoice;
 use std::time::{Duration, SystemTime};
 
 // TODO remove dead code after breez sdk implementation
```

Comment:
// TODO remove dead code after breez sdk implementation
#[allow(dead_code)]

These 2 lines can be removed, right?

Reply:
Done! Ty for noticing